kytos/core/api_server.py

```diff
@@ -1,12 +1,16 @@
 """Module used to handle a API Server."""
+import logging
 import os
-
 from urllib.error import URLError
 from urllib.request import urlopen
 
 from flask import Flask, request, send_from_directory
 from flask_socketio import SocketIO
 
+# OMIT API server logs
```

Comment:
1) Why omitting the API server logs?
2) Shouldn't this be a config from our controller (to be set on `kytos.conf`)?